float64[6] EndVel